### ISSUE TITLE
Making a prediction on Custom Data

### ISSUE THREAD

**Issue by N1h1l1sT** [unit8co/darts#1339]
I have a trained probabilistic TFTModel model.
The `input_chunk_length` is T, so it needs T previous timesteps.
To me, that means that to make 1 single-step prediction I would need a dataset `X` of NxTxD shape, where N is the observations, T is sequence length (number of previous timesteps) and D is the number of variables (1 for univariate, >1 for multivariate)

If I go `model.predict()`, I have to input a number of future predictions and it will use the train set as the previous values to predict the next ones. That is not what I want. I want to be able to predict on my own X inputs. How can I do that?

I have also set the `output_chunk_length` to 24, so I want 24 future values forecasted, which means the output for 1 observation of shape 1xTxD would yield a result of 1x24xK, where K is the dimensionality of my Targets.
So my target `Y` is of shape N x output_chunk_length x K

Basically, assume that I don't have the actual values (targets). I want to make a real forecast of actual future values that I can't have because they are in the future. I have T rows of my D variables available and the last row is now. How do I use those T rows to forecast the next 24 rows?

**Comment by dennisbader** [comment]
Hey @N1h1l1sT , if you haven't done so already I strongly recommend to read our [user guide on TorchForecastingModels](https://unit8co.github.io/darts/userguide/torch_forecasting_models.html).

All our GlobalForecastingModels such as TFTModel support prediction of any input target series with parameter `series` (and optional `past_covariates`, `future_covariates`). Here is also the documentation for [TFTModel's predict method](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.tft_model.html#darts.models.forecasting.tft_model.TFTModel.predict).

You can predict any numer of future time steps with parameter `n`, given that you have sufficient input data at prediction time.

Your `series` can either be either univariate (i.e. a list of `K` target series with only a single target component) or multivariate (i.e. a single target or multiple series each having `K` components (your dimensionality)). 
- You can train on a single target series at fitting time, but use a list of target series for prediction, or vice versa.
- Only the dimensionality of your target series must be the same for `fit()`/`predict()` (i.e. `K` components/columns).
